tgui/packages/tgui/interfaces/Fabrication/MaterialAccessBar.tsx

```diff
@@ -79,7 +79,7 @@ const MaterialCounter = (props: MaterialCounterProps, context) => {
     false
   );
 
-  const canEject = material.amount > 2_000;
+  const canEject = material.amount > 100;
```

Comment:
Can this be a define sent in static ui data perhaps?

Reply:
Can you describe in more detail so i know where to dig more presicely

Reply:
`SMALL_MATERIAL_AMOUNT`

Reply:
Static UI data would send the material define as data which is what this TGUI file can use instead of us having to manually change it when we edit material costs